interface/ceed-elemrestriction.c

```diff
@@ -357,6 +357,80 @@ int CeedElemRestrictionCreate(Ceed ceed, CeedInt num_elem, CeedInt elem_size,
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Create a CeedElemRestriction with orientation sign
+
+  @param ceed         A Ceed object where the CeedElemRestriction will be created
+  @param num_elem     Number of elements described in the @a offsets array
+  @param elem_size    Size (number of "nodes") per element
+  @param num_comp     Number of field components per interpolation node
+                        (1 for scalar fields)
+  @param comp_stride  Stride between components for the same L-vector "node".
+                        Data for node i, component j, element k can be found in
+                        the L-vector at index
+                        offsets[i + k*elem_size] + j*comp_stride.
+  @param l_size       The size of the L-vector. This vector may be larger than
+                        the elements and fields given by this restriction.
+  @param mem_type     Memory type of the @a offsets array, see CeedMemType
+  @param copy_mode    Copy mode for the @a offsets array, see CeedCopyMode
+  @param offsets      Array of shape [@a num_elem, @a elem_size]. Row i holds the
+                        ordered list of the offsets (into the input CeedVector)
+                        for the unknowns corresponding to element i, where
+                        0 <= i < @a num_elem. All offsets must be in the range
+                        [0, @a l_size - 1].
+  @param orient       Array of shape [@a num_elem, @a elem_size] with bool false
+                        for positively oriented and true to flip the orientation.
+  @param scale        An scalar value that scales the dofs in assembly.
+  @param[out] rstr    Address of the variable where the newly created
```

Comment:
Unrelated note - when we use vs omit in/out doesn't seem to have much rhyme or reason, we should be more consistent

Reply:
Sorry I don't understand what you mean?!

Reply:
That's more a global note for all of libCEED. For some documentation blocks we label all of our variables in or out, and for others we just label some, and for still others yet we label none. There isn't really any consistent rule to when we omit the in/out label that I can see.

Reply:
I see, yeah, I copied from the `CeedElemRestrictionCreate` and that's why I  have `out` here.